src/styles.css

```diff
@@ -31,6 +31,7 @@
   --bottom-bar-height: 80px;
   --primary-color: var(--blue-denim-400);
   --container-image-small-height: 250px;
+  container: quiz / inline-size;
```

Comment:
Add note about polyfill to README

Reply:
What is this sorcery sir?

Reply:
What is this sorcery sir?

Reply:
It's magic. Responded on Slack with details